monkey/infection_monkey/monkey.py

```diff
@@ -154,14 +164,13 @@ def _connect_to_island(self):
             logger.debug(f"Default server set to: {self._control_client.server_address}")
         else:
             raise Exception(
-                f"Failed to connect to the island via "
-                f"any known server address: {self._opts.servers}"
+                f"Failed to connect to the island via " f"any known server address: {self.server}"
             )
 
         self._control_client.wakeup(parent=self._opts.parent)
 
     def _current_server_is_set(self) -> bool:
-        if self._control_client.find_server(default_tunnel=self._opts.servers):
+        if self.server:
             return True
 
         return False
```

Comment:
I don't think we need this function anymore. If `find_server()` doesn't find any servers, we should exit. We shouldn't do nothing and then later check if it was successful.

If we get rid of `_current_server_is_set()`, then we can also get rid of `self.server`.